sdk/provisioning/Azure.Provisioning/src/signalr/SignalRService.cs

```diff
@@ -0,0 +1,62 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using Azure.Core;
+using Azure.Provisioning.ResourceManager;
+using Azure.ResourceManager.SignalR;
+using Azure.ResourceManager.SignalR.Models;
+
+namespace Azure.Provisioning.SignalR
+{
+    /// <summary>
+    /// Represents a SignalR.
+    /// </summary>
+    public class SignalRService : Resource<SignalRData>
+    {
+        private const string ResourceTypeName = "Microsoft.SignalRService/signalR";
+        private static readonly Func<string, SignalRData> Empty = (name) => ArmSignalRModelFactory.SignalRData();
+        internal const string DefaultVersion = "2023-02-01";
+
+        /// <summary>
+        /// Creates a new instance of the <see cref="SignalRService"/> class.
+        /// </summary>
+        /// <param name="scope">The scope.</param>
+        /// <param name="sku">The SKU.</param>
+        /// <param name="allowedOrigins">The allowed origins.</param>
+        /// <param name="serviceMode">The service mode.</param>
+        /// <param name="parent">The parent.</param>
+        /// <param name="name">The name.</param>
+        /// <param name="version">The version.</param>
+        /// <param name="location">The location.</param>
+        public SignalRService(IConstruct scope, SignalRResourceSku? sku = default, IEnumerable<string>? allowedOrigins = default, string serviceMode = "Default", ResourceGroup? parent = default, string name = "signalr", string version = DefaultVersion, AzureLocation? location = default)
+            : this(scope, sku, parent, name, false, (name) => ArmSignalRModelFactory.SignalRData(
+                name: name,
+                location: location ?? Environment.GetEnvironmentVariable("AZURE_LOCATION") ?? AzureLocation.WestUS,
+                sku: sku ?? new SignalRResourceSku("Free_F1") { Capacity = 1 },
```

Comment:
Is there a really constrained quota on free instances?  We might want to use the lowest paid option if so as the default in the CDK.

Reply:
Yeah you are limited to 1 unit

Reply:
Discussed offline - the defaults will need to be reviewed across the board anyway and we want to make sure that Aspire is not relying on the defaults in general.

Reply:
/cc @mitchdenny